packages/node/src/lib/organizations/organizations.interface.ts

```diff
@@ -0,0 +1,34 @@
+export interface IOrganizations {
+  list();
+  create(payload: IOrganizationCreatePayload);
```

Comment:
Why these two methods are public? For me it doesn't make sense - API KEY is attached to single organization, so why we want to allow to cross that boundary?

Whose organizations will be returned? Organizations of the owner?

I'm just wondering - but I implemented that anyway.